airbyte-integrations/connectors/source-redshift/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/RedshiftSourceAcceptanceTest.java

```diff
@@ -40,18 +41,28 @@ protected static JsonNode getStaticConfig() {
 
   @Override
   protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
-    config = getStaticConfig();
+    final JsonNode plainConfig = getStaticConfig();
```

Comment:
nitpicky, but I think this can be a bit simpler:
* make `getStaticConfig` return an ObjectNode (you can just make it `return (ObjectNode) Jsons.deserialize...`)
* don't use the new plainConfig variable. Replace the big `Jsons.jsonNode(ImmutableMap...)` call with `config.put("schemas", List.of(schemaName))`

This makes it easier to modify the config file (e.g. if you add a new config option, you won't need to remember to copy it from `plainConfig` back into `config`)

Reply:
Updated, thanks